### PR TITLE
OSDOCS#10114: Update hosted control planes with a newly available non bare metal (agent) platform

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.15+

Issue: https://issues.redhat.com/browse/OSDOCS-10114

Link to docs preview: https://73985--ocpdocs-pr.netlify.app/openshift-enterprise/latest/hosted_control_planes/hcp-configuring#hcp-configuring-non-bm-agent

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

File: hosted_control_planes/hcp-configuring.adoc [hunk 1]
```diff
@@ -13,36 +13,36 @@ You can view the procedures by selecting from one of the following providers:
...
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#hosting-cluster-aws-infra-reqs[AWS infrastructure requirements]: Review the infrastructure requirements to create a hosted cluster on {aws-short}.
+* link:https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.10/html/clusters/cluster_mce_overview#hosting-service-cluster-configure-aws[Configuring hosted control plane clusters on AWS]: The tasks to configure hosted control plane clusters on {aws-short} include creating the {aws-short} S3 OIDC secret, creating a routable public zone, enabling external DNS, enabling {aws-short} PrivateLink, and deploying a hosted cluster.
```
Comment (cbippley) [inline hosted_control_planes/hcp-configuring.adoc:45]: You've specified technology preview in other links so I would do so here as well.

**Review by cbippley — COMMENTED** [review]
One suggestion, otherwise LGTM!

**Review by opayne1 — COMMENTED** [review]
Just one more nit. Feel free to ping me and I can merge when you are done taking a look. 😄

**Review by opayne1 — COMMENTED** [review]
/lgtm

**Pull request merged**